src/main/java/frc/robot/config/Config.java

```diff
@@ -0,0 +1,75 @@
+package frc.robot.config;
+
+import edu.wpi.first.wpilibj.DriverStation;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class Config {
+
+    /**
+     * Instructions for set up of robot.conf file on robot
+     * <p>
+     * 1. Using a tool like Git Bash or putty, ssh into admin@roboRIO-2706-FRC.local
+     * a. There is no password on a freshly flashed roboRIO
+     * 2. Go up a directory (cd ..)
+     * 3. Create a new file called robot.conf (touch robot.conf)
+     * 4. Open the file with vi (vi robot.conf)
+     * 5. Press i to enter insert mode
+     * 6. Add an integer denoting the robot id. If it's the first robot, use 0, second use 1 etc.
+     * 7. Press [ESC] followed by typing :wq in order to save and quit
+     * 8. To verify this worked type: more robot.conf
+     * 9. If it displays the value you entered, it was successful
+     * 10. Type exit to safely exit the ssh session
+     */
+
+    private static final Path ROBOT_ID_LOC = Paths.get(System.getProperty("user.home"), "robot.conf");
+
+    /**
+     * ID of the robot that code is running on
+     */
+    private static int robotId = -1;
+
+    /**
+     * PLACE IDS OF ROBOTS HERE
+     **/
+
+    // Mergio is has the ID 2
+    private Config() {
+
+    }
+
+    /**
+     * Returns one of the values passed based on the robot ID
+     *
+     * @param first The first value (default value)
+     * @param more  Other values that could be selected
+     * @param <T>   The type of the value
+     * @return The value selected based on the ID of the robot
+     */
+    @SafeVarargs
+    private static <T> T robotSpecific(T first, T... more) {
+        if (getRobotId() < 1 || getRobotId() > more.length) {
+            return first;
+        } else {
+            return more[getRobotId() - 1];
+        }
+    }
+
+
+    private static int getRobotId() {
+        if (robotId < 0) {
+            try (BufferedReader reader = Files.newBufferedReader(ROBOT_ID_LOC)) {
+                robotId = Integer.parseInt(reader.readLine());
+            } catch (IOException | NumberFormatException e) {
+                robotId = 0;
+                DriverStation.reportError("Could not find robot configuration file.", false);
```

Comment:
Suggest something like "Could not find robot configuration file. Defaulting to robotId=0" so it's clear in the logs if we're ever debugging. Maybe also output the value of ROBOT_ID_LOC so the logs clearly show where and what it's looking for.

Reply:
In order to do this, we would first need to complete the logging functionality, to begin with. Should that be done before we merge the PR or can I just add it after?